apps/prenumerera/src/Package/Description.purs

```diff
@@ -87,11 +87,12 @@ packageDescriptions = fromFoldable
   , "HBL WEBB" /\
       { brand: HBL
       , brandLong: "HBL"
-      , descriptionShort: " Premium"
+      , descriptionShort: " Digital"
       , descriptionLong: description
-          [ Tuple "Premium" "Alla artiklar på hbl.fi"
-          , Tuple "Nyhetsappen HBL Nyheter" "Nyheter på mobilen och surfplattan, pushnotiser"
-          , Tuple "Digitalt månadsbrev" "Nyheter & förmåner"
+          [ Tuple "Premium" "Alla artiklar på HBL.fi, Vastranyland.fi, Ostnyland.fi"
```

Comment:
This doesn't seem to match the sheet. https://docs.google.com/spreadsheets/d/16YeLA-wpKzncz9P3dpGdeWp9I8oMhOKrjDjalS58Qpo/edit#gid=0
`Alla artiklar på HBL.fi`
And the next line `HBL Nyheter (iOS & Android)`